### PR TITLE
Updates to SaaS API Getting started page

### PR DESCRIPTION
**Pull request by MKhalusova**
**DO NOT MERGE UNTIL API LAUNCH**

This PR includes updates to the Getting Started docs for the new API (linked from the dashboard).

### CODE REVIEW

**Review by ninalopatina — APPROVED** [review]

File: api-reference/api-services/saas-api-development-guide.mdx [hunk 1]
```diff
@@ -86,7 +82,7 @@ from unstructured_client.models import operations, shared
 
 client = unstructured_client.UnstructuredClient(
     api_key_auth="YOUR_API_KEY",
-    # server_url="YOUR_API_URL",
+    server_url="YOUR_API_URL",
```
Comment (amanda103) [inline api-reference/api-services/saas-api-development-guide.mdx:85]: can we call this something other than 'YOUR_API_URL'? this makes me think that we have unique url like in legacy. Maybe just 'API_URL'?

File: api-reference/api-services/saas-api-development-guide.mdx [hunk 2]
```diff
@@ -167,51 +163,16 @@ filename = "PATH_TO_FILE"
 elements = partition_via_api(
   filename=filename,
   api_key="YOUR_API_KEY",
-  api_url="https://YOUR_API_HOSTNAME.api.unstructuredapp.io/general/v0/general"
+  api_url="YOUR_API_URL"
```
Comment (amanda103) [inline api-reference/api-services/saas-api-development-guide.mdx:166]: Same thing here on this var name^

File: api-reference/api-services/saas-api-development-guide.mdx (hunk 1 above)
Comment (MKhalusova) [reply api-reference/api-services/saas-api-development-guide.mdx:85]: Sure :)

**Comment by cmscmadd** [comment]
LGTM Thanks MK

**Comment by cmscmadd** [comment]
LFG

**Pull request merged**